build_web_compilers/lib/builders.dart

```diff
@@ -96,12 +98,19 @@ Map<String, String> _readEnvironmentOption(BuilderOptions options) {
   return Map.from((options.config[_environmentOption] as Map) ?? {});
 }
 
+List<String> _readExperimentOption(BuilderOptions options) {
+  return List<String>.from(
```

Comment:
[nit] the `<String>` isn't necessary here since it gets inferred from the return type.

Reply:
for some reason I was thinking this was an implicit cast and not type inference :P